fabrication.py

```diff
@@ -393,3 +393,25 @@ def generate_bom(self):
         self.logger.info(
             "Finished generating BOM file %s", os.path.join(self.outputdir, bomname)
         )
+
+    def check_plausibility(self) -> str:
```

Comment:
Name is ok but wondering if 'get_part_consistency_warnings' or something would be more descriptive (thought of this after looking at the next commit).